Add external zones article (#1032)
dnsimple/dnsimple-support: content/articles/external-dns-providers.markdown

---
title: External DNS Providers
excerpt: Link an external DNS provider to your DNSimple account to manage zones at other authoritative DNS providers, all within DNSimple.
categories:

> - DNS

Reviewer: I wonder if this should also be in `Integrations`.

Author: I think that's a great idea! Added it: https://github.com/dnsimple/dnsimple-support/pull/1032/commits/063a34d9040a1d9f1bbc0274f3e416404e633b69

## Supported External DNS Providers

In order to manage [external zones](/articles/managing-external-zones) from DNSimple, you first have to link an external DNS provider to your DNSimple account.

Currently, DNSimple supports the management of zones from the below authoritative DNS providers.

### Amazon Route 53

DNSimple supports [Amazon Route 53](https://aws.amazon.com/route53/) as an external DNS provider.

#### Prerequisites for linking Amazon Route 53 as an external DNS provider

To proceed with linking [Amazon Route 53](https://aws.amazon.com/route53/) as an external DNS provider to your account, you must:

- Have an [AWS access key](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html) for an AWS user with permission to manage public hosted zones at Route 53.
- Have administrator access to a DNSimple account

## Supported record types {#supported-record-types}

Records of supported record types can be [synced](/articles/deployment-editor#record-syncing) from/to DNSimple to/from an external DNS provider.

> However, record fields that are not supported in DNSimple records will not be synced. Likewise, record fields like [record notes](/articles/record-notes) and [regional record](/articles/regional-records) information are only supported in DNSimple records and will not be synced to the external DNS provider.

Reviewer: ```suggestion
However, record fields that are not supported by the destination provider will be ignored. Likewise, record fields like [record notes](/articles/record-notes) and [regional record](/articles/regional-records) information are only supported by DNSimple and will not be synced to the external DNS provider.
```

Author: What do you think of `record fields that are not supported by DNSimple, or the destination provider` ?

Reviewer 2: @dallasread Made the change: https://github.com/dnsimple/dnsimple-support/pull/1032/files/ec05b9231e50ff9f885b25711558094ded0be737..a775c9357ef914c11a8fb1035aa2ed1dbd69710b (cc @itsalyse)

Author: @dallasread Made the change: https://github.com/dnsimple/dnsimple-support/pull/1032/files/ec05b9231e50ff9f885b25711558094ded0be737..a775c9357ef914c11a8fb1035aa2ed1dbd69710b (cc @itsalyse )

- **Management of external zone records**: List, create, update, and delete external zone records from DNSimple using the [Deployment Editor](/articles/deployment-editor). (Note: The list of supported record types differs based on the provider)
- **2-way Syncing of Records**: Sync your external zone records from the external zone to DNSimple, or from DNSimple to the external zone, with the [Deployment Editor](/articles/deployment-editor#record-syncing).

## Supported External DNS Providers

To manage [external zones](/articles/managing-external-zones) from DNSimple, you first have to link an external DNS provider to your DNSimple account.

DNSimple supports the management of zones from the authoritative DNS providers listed below.

### Amazon Route 53

DNSimple supports [Amazon Route 53](https://aws.amazon.com/route53/) as an external DNS provider.

#### Prerequisites for linking Amazon Route 53 as an external DNS provider

To link [Amazon Route 53](https://aws.amazon.com/route53/) as an external DNS provider, you need:

> - An [AWS access key](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html) for an AWS user with permission to manage public hosted zones at Route 53.

Reviewer: Should we list the minimum permissions required? https://github.com/dnsimple/dnsimple-ops/issues/3193#issue-1710471000

Author: Added in https://github.com/dnsimple/dnsimple-support/pull/1032/commits/7c4857bd0a477c938380de52e06cdaec32b90d3d 🎉
